kernel/kernel-api/src/main/java/io/delta/kernel/internal/skipping/DataSkippingUtils.java

```diff
@@ -364,11 +408,14 @@ private static Optional<Predicate> constructNotDataSkippingFilters(
                         schemaHelper.isSkippingEligibleLiteral(rightLit)) {
                         // Match any file whose min/max range contains anything other than the
                         // rejected point.
-                        // For example a = 1 --> minValue.a <= 1 AND 1 <= maxValue.a
+                        // For example a != 1 --> minValue.a < 1 OR maxValue.a > 1
```

Comment:
why did you change the comment for the case `=` to be for `!=`?

Reply:
It was wrong before this fixes it. This is in the constructNotDataSkippingFilter so it's NOT(=) --> !=